java-backend/src/main/java/com/project/spring/Endpoint/UnequalPayEndpoint.java

```diff
@@ -0,0 +1,43 @@
+package com.project.spring.Endpoint;
+
+import com.google.gson.Gson;
+import com.project.spring.Services.UnequalPayLookupService;
+import io.swagger.annotations.ApiOperation;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.ArrayList;
+
+@RestController
+@RequestMapping(value = "/api/unequalPayLookup")
+public class UnequalPayEndpoint {
+
+    @Autowired
+    private UnequalPayLookupService unequalPayLookupService;
+
+    final static Logger logger = LogManager.getLogger(UnequalPayEndpoint.class);
+
+    @ApiOperation(value = "lists all of the employees that make less than their juniors")
+    public ResponseEntity<String> findUnequallyPaidEmployees() {
```

Comment:
Using a response entity that is a string is not that great, because it's not clear to the client/caller of the endpoint what the format of the payload should be. You're basically making them guess how the string is formatted (such as xml, json, etc). There are a couple ways to handle this, but I would recommend using some of the fancy annotations that Spring offers.